xslt/Image.xsl

```diff
@@ -38,18 +48,21 @@
 
     <!-- /ImageDocument/Metadata/Information/Image/Dimensions/S/Scenes/Scene => /OME/Image/Image -->
     <xsl:template match="Scene">
+        <xsl:variable name="idx" select="@Index"/>
         <xsl:element name="ome:Image">
             <xsl:attribute name="ID">
                 <xsl:text>Image:</xsl:text>
-                <xsl:value-of select="@Index"/>
+                <xsl:value-of select="$idx"/>
```

Comment:
I'm just curious here is the motivation for creating the $idx parameter so you can set an index you want or as it needed because the @Index value wasn't correct?

Reply:
`$idx` is needed in the `common_image_contents` and `Image` templates to translate data from subblocks and channels for the appropriate scene.